tests/urdf_examples/pelican.urdf

```diff
@@ -0,0 +1,108 @@
+<robot name="Asctec Pelican">
```

Comment:
If you copied a urdf from somewhere you should properly attribute the owner and check that their licence permits this copy

Reply:
Also what purpose does this urdf serve? It is not called from anywhere